core/auth/example/config/config.cue

```diff
@@ -1,5 +1,23 @@
 AuthHttp1 :: core.auth.http & {broker: "http1", realm: "http1 realm", users: {"user1": "pw1"}}
 AuthHttp2 :: core.auth.http & {broker: "http2", realm: "http2 realm", users: {"user2": "pw2"}}
+AuthFake1 :: core.auth.fake & {broker: "fake1", userConfig: {jondoe: {password: "password"}}}
+AuthFake1WithoutPasswords :: core.auth.fake & {broker: "fake1WithoutPasswords", validatePassword: false, userConfig: {jondoe: {}}}
+AuthFake1WithDefaultTemplate :: core.auth.fake & {broker: "fake1WithDefaultTemplate", validatePassword: true, userConfig: {jondoe: {password: "password"}}}
+AuthFake1WithCustomTemplate :: core.auth.fake & {broker: "fake1WithCustomTemplate", validatePassword: true, usernameFieldId: "customUsernameField", passwordFieldId: "customPasswordField", userConfig: {jondoe: {password: "password"}}, loginTemplate: """
+<body>
+  <h1>Custom Login Template!</h1>
+  <form name="fake-idp-form" action="{{.FormURL}}" method="post">
+	<div>{{.Message}}</div>
+	<label for="{{.UsernameID}}">Username</label>
+	<input type="text" name="{{.UsernameID}}" id="{{.UsernameID}}">
+	<label for="{{.PasswordID}}">Password</label>
+  <input type="password" name="{{.PasswordID}}" id="{{.PasswordID}}">
+	<label for="{{.OtpID}}">2 Factor OTP</label>
```

Comment:
`{"code":500,"error":"template: fake:9:15: executing \"fake\" at \u003c.OtpID\u003e: can't evaluate field OtpID in type fake.viewData"}`